drivers/SmartThings/matter-button/src/init.lua

```diff
@@ -1,3 +1,8 @@
+-- As of July 2024, the matter-button driver has been deprecated in favor of being integrated into the matter-switch
+-- driver. All functionality and test cases present in this driver were carried over into matter-switch. Note that this
+-- won't affect devices using the button driver unless they are re-onboarded, in which case they would onboard to the
+-- switch driver, as all button fingerprints were removed from this driver and moved to the switch driver.
+
```

Comment:
I would add additionally here that we are no longer accepting bug fixes, feature enhancements such as fingerprint adds to this driver. 

Something along those lines to make it really clear - there are no planned changes to folks who might be reading this.

Reply:
Updated the comment as suggested!